docs/src/main/paradox/extensions.md

```diff
@@ -1,14 +1,12 @@
 # 6. Extensions
 
-There are several third party libraries that expand the functionality of Akka Http.
+There are several [third party libraries](https://doc.akka.io/docs/akka-http/10.2/extensions.html) that expand the functionality of Akka HTTP.
 
-Among those, we want to highlight the following:
+Some that support Apache Pekko HTTP snapshots include:
 
-- [akka-http-json](https://github.com/hseeberger/akka-http-json): Integrate some of the best JSON libs in Scala with Akka HTTP
```

Comment:
Can we make a note of these extensions so we can speak to upstream about whether they can move to pekko?

Reply:
I have opened a conversation in Heiko Seeburger's project. He want volunteers to help out. This might be the way things go. I'm a little reluctant to be the only volunteer and end up maintaining Akka libs as well as the Pekko ones.